lib/travis/api/v3/routes.rb

```diff
@@ -51,6 +51,11 @@ module Routes
       end
     end
 
+    resource :installation do
+      route '/installation/{installation.github_id}'
+      get :find
+    end
```

Comment:
This looks good. I don't think an owner specific route is needed, as with the suggested change to the installation renderer the response will already include the owner object.